ansible/roles_ocp_workloads/ocp4_workload_authentication/tasks/workload.yml

```diff
@@ -12,7 +15,10 @@
   - name: Generate cluster admin password
     when: ocp4_workload_authentication_htpasswd_admin_password | default('') | length == 0
     set_fact:
-      ocp4_workload_authentication_admin_password: "{{ lookup('password', '/dev/null length={{ ocp4_workload_authentication_htpasswd_admin_password_length }} chars=ascii_letters,digits') }}"
+      ocp4_workload_authentication_admin_password: >-
+        {{ lookup('password',
+        '/dev/null length={{ ocp4_workload_authentication_htpasswd_admin_password_length }}
+        chars=ascii_letters,digits') }}
```

Comment:
Hrm, this looks really weird... The way lookup plugins take arguments is weird... but how about:

```
      ocp4_workload_authentication_admin_password: >-
￼        {{ lookup('password', '/dev/null chars=ascii_letters,digits '
￼            ~ 'length=' ~ ocp4_workload_authentication_htpasswd_admin_password_length
￼        ) }}
```

Reply:
It doesn't seem to work.
syntax error: could not find expected ':' (syntax) is the error I'm receiving

Reply:
Never mind, figured it out.